.config/.env.example

```diff
@@ -1,5 +1,6 @@
 # Database Config
-MONGODB_USER=development
-MONGODB_PASSWORD=
-MONGODB_DATABASE=development
-MONGODB_URI=mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@mongodb:27017/${MONGODB_DATABASE}
+POSTGRES_HOST=
+POSTGRES_PORT=
+POSTGRES_DB=db_dev
+POSTGRES_USER=admin
+POSTGRES_PASSWORD=development
```

Comment:
Obviously this is the example and not used anywhere, but I still prefer to leave password fields blank :) I usually provide an example dummy value for everything but passwords

Reply:
Good point, changing it.